test/test_meta.py

```diff
@@ -968,6 +1058,45 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
             run_symbolic_meta=self.symbolic_meta,
         )
 
+        # This is to test torch ops that do not have an out parameter but have
+        # aten op overloads that have out parameters. Additionally, Python decompositions
+        # may register OpOverloadPacket's so decompositions need to be tested
+        # to ensure all OpOverloads still function for the Meta key (e.g. if a python decomposition
+        # is registered for an aten op aten.foo with overloads [default, out], the python
+        # function needs to support receiving `out` arguments)
+        if (
+            not self.inplace and
+            not self.supports_out and
+            (torch.is_tensor(expected) or isinstance(expected, Iterable))
+        ):
```

Comment:
Right. In that case, we should probably just test this if the default function is expected to pass, otherwise we end up testing the same xfail twice really. We have very little hope for the out test to pass when the default function does not pass.